README.md

```diff
@@ -22,7 +22,7 @@ If you do not have access to a RHMAP instance, you can sign up for a free instan
 
 This source repository may be imported into any project by using the "Import Existing App" feature when you add a new client app to a project.  It may be imported into any project.
 
-**If you do not name your AuthPolicy "Google" you will have to edit [FHOauth](app/src/main/java/com/feedhenry/oauth/oauth_android_app/FHOAuth.java) and set "FH_AUTH_POLICY" field to your auth policy's name.**
+**If you do not name your AuthPolicy "Google" you will have to edit `FHOAuth.java` and set "FH_AUTH_POLICY" field to your auth policy's name.**
```

Comment:
I really prefer what you did in https://github.com/feedhenry-templates/pushstarter-android-app/pull/26 using the file path instead of only the file name. Wdyt?